mcs/class/System/Test/System.Net.NetworkInformation/IPInterfacePropertiesTest.cs

```diff
@@ -0,0 +1,73 @@
+//
+// IPInterfacePropertiesTest.cs - NUnit Test Cases for System.Net.NetworkInformation.IPInterfaceProperties
+//
+// Authors:
+//   Ben Woods (woodsb02@gmail.com)
+//
+
+using NUnit.Framework;
+using System;
+using System.Net;
+using System.Net.NetworkInformation;
+
+namespace MonoTests.System.Net.NetworkInformation
+{
+
+[TestFixture]
+public class IPInterfacePropertiesTest
+{
+	[Test]
+	public void AtLeastOneUnicastAddress ()
+	{
+		int numUnicastAddresses = 0;
+		NetworkInterface[] adapters = NetworkInterface.GetAllNetworkInterfaces();
```

Comment:
Please use a space before the opening parenthesis, as shown in the [coding guidelines](http://www.mono-project.com/community/contributing/coding-guidelines/).

This applies to the rest of the tests as well.